src/book/03-guides/01-main-codebase/02-script-tuner.mdx

```diff
@@ -0,0 +1,99 @@
+| <kbd>R</kbd>                   | Manual refresh view.                                                                                                                                                                                                                                                                                                                                                                                 |
+| <kbd>M</kbd>                   | Mirrors the whole script.                                                                                                                                                                                                                                                                                                                                                                            |
+| <kbd>X</kbd>                   | Exits the help or exits gain editing.                                                                                                                                                                                                                                                                                                                                                                |
+| <kbd>CTRL</kbd> + <kbd>C</kbd> | Quits script tuner.                                                                                                                                                                                                                                                                                                                                                                                  |
+| <kbd>:help</kbd>               | Lists all commands that can be used in script tuner.                                                                                                                                                                                                                                                                                                                                                 |
+
+## List of Scripts
+
+| Script Name          | Description                                                              |
+| -------------------- | ------------------------------------------------------------------------ |
+| `KickLeft.yaml`      | Robot kicks with its left foot.                                          |
+| `KickRight.yaml`     | Robot kicks with its right foot.                                         |
+| `NodYes.yaml`        | The robot nods its head.                                                 |
+| `Relax.yaml`         | The robot relaxes all of its joints.                                     |
```

Comment:
```suggestion
| `Relax.yaml`         | The robot relaxes all of its joints. Be prepared to hold it if it's standing, as it might fall over.                             |
```